ui/panels-plugin/src/plugins/empty-chart/empty-chart-model.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { JsonObject } from '@perses-dev/core';
+
+export type EmptyChartOptions = JsonObject;
```

Comment:
I like the having explicit name for this type (even tho it's just JsonObject) and the explict model file!